### ISSUE TITLE
Build instructions are not correct

### ISSUE THREAD

**Issue by carlesfernandez** [acebrianjuan/gnss-sdr-monitor#5]
Hi,

I tried to build de software in a fresh Ubuntu 18.04 installation.

- Instructions are not right (e.g., `cd gnss-sdr/build` does not exist)

- There is no an install target (so I can't install the application system wide).

Could you please fix theses issues?

**Comment by acebrianjuan** [comment]
Dear @carlesfernandez,

There was indeed an error in the building instructions, where it should say `cd gnss-sdr-monitor/build`.
I have fixed this in commit 726323c.

Regarding the install target for CMake, I have created one and submitted it in commit af9b450.
The results of the latest GitLab CI/CD pipeline [#84663092](https://gitlab.com/acebrianjuan/gnss-sdr-monitor/pipelines/84663092) show that the `make install` rule for the gnss-sdr-monitor GUI works correctly for Ubuntu 18.04 and above.

Thank you for reporting this issue.

Regards,
Álvaro
